libs/server/features/src/security-pricing/security-pricing.service.ts

```diff
@@ -73,10 +78,15 @@ export class SecurityPricingService implements ISecurityPricingService {
     }
 
     async syncAll() {
```

Comment:
This function syncs the pricing of all securities. Could it be changed to something that's reflective of that (same with the `sync` function above this? I got confused and thought it might be syncing all stocks because the next function is called `syncUSStockTickers`